### PR TITLE
fix(curriculum): typos find by crowdin

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

these are typos found in crowding

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/take-home-projects/show-the-local-weather.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ dashedName: show-the-local-weather
 
 # --description--
 
-**Objective:** Build a an app that is functionally similar to this: <https://codepen.io/freeCodeCamp/full/bELRjV>.
+**Objective:** Build an app that is functionally similar to this: <https://codepen.io/freeCodeCamp/full/bELRjV>.
```
Comment (ieahleen) [inline curriculum/challenges/english/10-coding-interview-prep/take-home-projects/show-the-local-weather.md:11]: shouldn't this link also be converted?

```suggestion
**Objective:** Build an app that is functionally similar to this: <a href="https://codepen.io/freeCodeCamp/full/bELRjV" target
="_blank" rel="noopener noreferrer nofollow">https://codepen.io/freeCodeCamp/full/bELRjV</a>.
```

Comment (Sboonny) [reply curriculum/challenges/english/10-coding-interview-prep/take-home-projects/show-the-local-weather.md]: Thank you, haven't noticed it💚

**Review by majestic-owl448 — DISMISSED** [review]

**Comment by gikf** [comment]
Could you explain why changing element(s) to item(s) is needed?

**Comment by Sboonny** [comment]
@gikf in translation elements are used to describe `html` tags. for example, change a `fieldset` element, or indent `p` and `h1` elements. 

So using elements here lead to inconsistency in the curriculum, and maybe a misunderstanding. But Rosette challenge are advance challenges, so the change is quality of life and unneeded.

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/hash-from-two-arrays.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ dashedName: hash-from-two-arrays
 
 # --description--
 
-Using two arrays of length, create a hash object where the elements from one array (the keys) are linked to the elements of the other (the values).
+Using two arrays' length, create a hash object where the elements from one array (the keys) are linked to the elements of the other (the values).
```
Comment (moT01) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/hash-from-two-arrays.md:11]:
```suggestion
Given two arrays, create a hash object where the elements from one array (the keys) are linked to the elements of the other (the values).
```
This doesn't quite make sense to me. It seems like this is saying to use the length of the arrays - which you do, but not in the context of this sentence.

**Review by moT01 — APPROVED** [review]
I'm fine with these changes. Element and item seem pretty interchangeable to me. I would have stuck with element.

**Review by majestic-owl448 — APPROVED** [review]

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/hash-from-two-arrays.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ dashedName: hash-from-two-arrays
 
 # --description--
 
-Using two arrays of length, create a hash object where the elements from one array (the keys) are linked to the elements of the other (the values).
+Given two arrays, create a hash object where the elements from one array (the keys) are linked to the elements of the other (the values).
```
[the remark this answers is not in the document]
Comment (ieahleen) [reply curriculum/challenges/english/10-coding-interview-prep/rosetta-code/hash-from-two-arrays.md:11]: oh, alright

**Review by majestic-owl448 — APPROVED** [review]

**Pull request merged**